tardis/plasma/base.py

```diff
@@ -58,8 +56,10 @@ def _build_graph(self):
         """
         Builds the directed Graph using network X
 
-        :param plasma_modules:
-        :return:
+        Parameters
+        ----------
+        ~plasma_properties : list
```

Comment:
Why is `~` needed here?

Reply:
I thought it is used for parameters that are not passed as arguments, but i didn't find any docs to support that. Should i remove it?